gflownet/gflownet.py

```diff
@@ -57,6 +53,7 @@ def __init__(
         sample_only=False,
         replay_sampling="permutation",
         train_sampling="permutation",
+        eval_config=None,
         **kwargs,
     ):
         # Seed
```

Comment:
Should we remove references to `oracle` across the code? Everywhere I look it seems deprecated / unused.

Reply:
Yes, we should. We could decide later on to bring it back at some point, but that decision would imply wide changes. So yes, we can safely and should remove all references to oracles.

Reply:
Sounds good

Reply:
Do you also want to remove:

1. `gflownet/oracle/*`
2. `scripts/oracle_annotate.py`
3. `scripts/oracle_sampler.py`
4. Is `energies = env.oracle(x_sampled)` in `eval_gflownet.py` still relevant?

Reply:
1-3 can go.

4 is actually wrong :/ and I thought it had been corrected but I guess it is corrected only in another fork... Keep this and I will take care of this myself. The entire script needs to be updated.

Reply:
Done in [`82cc7ec` (#287)](https://github.com/alexhernandezgarcia/gflownet/pull/287/commits/82cc7ecb77532a4e94f8d805089b72c91e6b53ea)

NB: I also removed `legacy.py`. There are also a few remaining `oracle` references, for instance `proxy_state_format: oracle`.

Reply:
I'll let you resolve when you've read it

Reply:
Done in [this PR](https://github.com/alexhernandezgarcia/gflownet/pull/290). Feel free to review that (very small), merge and also merge that branch into this one to get the changes. Agree?